packages/@ionic/cli-utils/src/definitions.ts

```diff
@@ -143,8 +143,12 @@ export interface ProjectCordovaIntegration extends ProjectIntegration {
   setupEngineHooks?: boolean;
 }
 
+export interface ProjectCapacitorIntegration extends ProjectIntegration {
+}
+
 export interface ProjectIntegrations {
   cordova?: ProjectCordovaIntegration;
+  capacitor?: ProjectCapacitorIntegration;
```

Comment:
You can put integration files here: https://github.com/ionic-team/starters/tree/master/integrations

Then you'll get a url for the `.tar.gz`, which you can use in these files: https://github.com/ionic-team/ionic-cli/tree/master/packages/%40ionic/cli-utils/src/lib/integrations